controllers/ipfs.go

```diff
@@ -103,14 +107,25 @@ func ListStreams(c *gin.Context) {
 	var response *P2PStreamsList
 	s := shell.NewLocalShell()
 	reqBuilder := s.Request("p2p/stream/ls")
-	// reqBuilder.Arguments("arg1", "arg2")
 	err := reqBuilder.Exec(context.Background(), &response)
 	if err != nil {
 		log.Fatal(err)
 	}
 	c.IndentedJSON(http.StatusOK, response.Streams)
 }
 
+func EnableLibp2pStreaming() {
+	var response *ConfigResult
+	s := shell.NewLocalShell()
+	reqBuilder := s.Request("config")
+	reqBuilder.Arguments("Experimental.Libp2pStreamMounting", "true")
+	reqBuilder.Option("bool", true)
```

Comment:
If the argument has already set to true, why do you need to set this to true again?

Reply:
Here is the Kubo RPC documentation http://docs.ipfs.tech.ipns.localhost:8080/reference/kubo/rpc/#api-v0-config

After some tests, we need to have both 

- arg [string]: The value to set the config entry to.    here --> "true"
- bool [bool]: Set a boolean value.                      here --> true

or it doesn't apply the config, just returns the current state.